sample/registry/clusterrole.yaml

```
@@ -0,0 +1,23 @@
+  - list
+  - watch
+- apiGroups:
+  - ml.llmos.ai
+  resources:
+  - registries
+  - models
+  - datasetversions
+  verbs:
+  - get
+  - list
+  - watch
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)